cohttp-eio/src/cohttp_eio.mli

```diff
@@ -96,3 +100,80 @@ module Server : sig
 
   val not_found_handler : handler
 end
+
+module Client : sig
+  type response = Http.Response.t * Eio.Buf_read.t
+
+  type host = string * int option
+  (** Represents a server host - as ip address or domain name - and an optional
+      port value, e.g. www.example.org:8080, www.reddit.com *)
+
+  type resource_path = string
+  (** Represents HTTP request resource path, e.g. "/shop/purchase",
+      "/shop/items", "/shop/categories/" etc. *)
+
+  type ('a, 'b) conn = 'a -> (resource_path * host * #Eio.Flow.two_way as 'b)
+  (** [('a, 'b conn)] is [(resource_path, host, flow)]. [flow] is the Eio flow
```

Comment:
```suggestion
  (** [('a, 'b) conn] is [(resource_path, host, flow)]. [flow] is the Eio flow
```
However, this description does not match the type, which is still (strangely) a function.

Reply:
I have removed `'a conn` now. The are now passed directly to `call` as parameters. 7599e46254af78cc176d7a982deb3815d95a191a